code/datums/traits.dm

```diff
@@ -483,6 +483,13 @@
 	points = -1
 	category = list("trinkets")
 
+/datum/trait/petperson
```

Comment:
It would be really nice (but not essential) if this had an icon for the trait screen.

Reply:
Yeah that would be sick, I asked some peeps, will see if something turns up, any sprite I would whip up would be mediocre